toolkit/tools/internal/logger/log.go

```diff
@@ -179,8 +209,15 @@ func ReplaceStderrFormatter(newFormatter logrus.Formatter) (oldFormatter logrus.
 	return stderrHook.ReplaceFormatter(newFormatter)
 }
 
-func initStderrLogInternal(callerFilePath string) {
-	const useColors = true
+func initStderrLogInternal(callerFilePath string, color string) {
+	var useColors bool
+	if color == colourModeAuto || color == colourModeAlways {
+		useColors = true
+	} else if color == colourModeNever {
+		useColors = false
+	} else {
+		useColors = true
+	}
```

Comment:
I think we can simplify this logic a bit:
```suggestion
	useColors := true
	if color == colourModeNever {
		useColors = false
	}
```

Also, please pick either "colour" or "colour" - let's be consistent across the code.:)

Reply:
Thank you for catching that inconsistency

Reply:
I was trying to be intentional about the cases, so any future changes would require attention here. But I can see how it looks messy, will change as you suggested